tutorials/eth-api/using-tenderly.md

```diff
@@ -0,0 +1,175 @@
+
+1. Expand the **State Overrides** section
+2. Press **Add State Override**
+3. Select the relevant contract, in this case the USDC one
+4. Under the **Storage Variables** section, we're going to override the mapping that holds the balance of Baltathar by specifying the key as: `balanceOf[0x3Cd0A705a2DC65e5b1E1205896BaA2be8A07c6e0]` and value as: `10000000000` Pay careful attention that you are performing this step in the **Storage Variables** section and not the **Balance** section
+5. Press **Add** to confirm adding the state override
+6. Press **Simulate Transaction**
+
+![Simulate a transaction against Moonbeam with state overrides](/images/tutorials/eth-api/using-tenderly/tenderly-4.png)
+
+!!! note
+    Remember that the Alith and Baltathar accounts are part of the [list of public developer accounts](/builders/get-started/networks/moonbeam-dev/#pre-funded-development-accounts){target=_blank} with known private keys. You will lose any funds sent to these addresses.
+
+If you correctly added the state override, you should now see a transaction simulation success screen upon running the simulation. If you get an error, you can press **Re-Simulate** and verify that you have configured the state override correctly. 
+
+![Transaction simulation with state override success](/images/tutorials/eth-api/using-tenderly/tenderly-5.png)
+
+You can also access Tenderly's transaction simulator via the [Tenderly Simulation API](https://docs.tenderly.co/simulations-and-forks/simulation-api){target=_blank}. 
+
+## Fork the Chain {: #fork-the-chain }
+
+### Create a Fork {: #create-a-fork }
+
+Simulations are great for one-off tests, but what if you want to test a series of interdependent transactions? A fork is a better option in this case because forks are stateful. Additionally, [Tenderly forks](https://docs.tenderly.co/simulations-and-forks/forks){target=_blank} are a terrific option when you want to interact with contracts in a private environment, without re-deploying existing smart contract logic on-chain. 
+
+!!! note
+    There are some limitations to be aware of when using Tenderly's forking feature. You cannot interact with any of the [Moonbeam precompiled contracts](/builders/pallets-precompiles/precompiles/){target=_blank} and their functions. Precompiles are a part of the Substrate implementation and therefore cannot be replicated in the simulated EVM environment. This prohibits you from interacting with cross-chain assets on Moonbeam and Substrate-based functionality such as staking and governance.
+
+It's easy to create a fork with Tenderly. To do so, take the following steps:
```

Comment:
users might still be on the simulator tab, so just a reminder to switch to forks would be nice